client/scripts/bundle.js

```diff
@@ -198,6 +199,17 @@ async function main() {
         "@nomicfoundation/solidity-analyzer-linux-x64-gnu": "0.1.1",
         "@nomicfoundation/solidity-analyzer-linux-x64-musl": "0.1.1",
         "@nomicfoundation/solidity-analyzer-freebsd-x64": "0.1.1",
+
```

Comment:
Same question here, I wonder why are we packaging all of these, given that `vsce` can detect dependencies automatically through `package.json`?

Reply:
I think this is here so the .vsix file includes all platform-specific packages and then the same  vsix can be used across all platforms.

Reply:
Just wondering, is this intended, or just a limitation of the current build?
My understanding is that `vsce` now supports platform-specific extensions natively:

https://code.visualstudio.com/api/working-with-extensions/publishing-extension#platformspecific-extensions

Reply:
A limitation of the current build.

I have opened an issue for the platform specific visx file approach: https://github.com/NomicFoundation/hardhat-vscode/issues/534

Reply:
Thanks for confirming. If we can still publish the extension with the added size, it is not a blocker then.

Reply:
I think we can, @antico5 can you check the difference in size?

Reply:
> Including all the platform packages (current): 17.5 Mb

I think that would work. I tested a few popular extensions in the marketplace and they are 30-50mb.
Thanks for checking!